dir.targets

```diff
@@ -154,5 +154,12 @@
 
   </Target>
 
+  <Target Name="SetPublicReleaseVersion" AfterTargets="GetBuildVersion">
+    <PropertyGroup Condition=" '$(PublicRelease)' == 'true' ">
+      <!-- Removes the prerelease tag from version when this is a public version but still has the git commit-->
+      <AssemblyInformationalVersion>$(BuildVersionSimple)$(SemVerBuildSuffix)</AssemblyInformationalVersion>
```

Comment:
I agree.  I think when we redo our projects, we should remove the prerelease tag from GitVersioning and add it ourselves if its not a public release.   It only applies to NuGet package anyway, I don't think it should show up in any assembly versions anyway.